### PR TITLE
feat: Updated constant files with supported cssClasses

### PR DESCRIPTION
**Pull request by abhiomkar**
Refs https://github.com/material-components/material-components-web-react/issues/827

by @TroyTae

### CODE REVIEW

File: packages/mdc-linear-progress/constants.ts [hunk 1]
```diff
@@ -22,9 +22,19 @@
...
 export const cssClasses = {
-  CLOSED_CLASS: 'mdc-linear-progress--closed',
-  INDETERMINATE_CLASS: 'mdc-linear-progress--indeterminate',
-  REVERSED_CLASS: 'mdc-linear-progress--reversed',
+  BAR: 'mdc-linear-progress__bar',
+  BAR_INNER: 'mdc-linear-progress__bar-inner',
+  BUFFER: 'mdc-linear-progress__buffer',
+  BUFFERING_DOTS: 'mdc-linear-progress__buffering-dots',
+  CLOSED: 'mdc-linear-progress--closed',
+  CLOSED_CLASS: 'mdc-linear-progress--closed', // deprecated
```
Comment (moog16) [inline packages/mdc-linear-progress/constants.ts]: if these are deprecated, shouldn't we just do a rename through the rest of the package?

Comment (TroyTae) [reply packages/mdc-linear-progress/constants.ts]: I want to notice about deprecated constants.
This is my original my code:
```
const cssClasses = {
  get CLOSED_CLASS() {
    console.warn('MDCWeb: CLOSED_CLASS is deprecated, use CLOSED instead');
    return 'mdc-linear-progress--closed';
  }
};
```
But mdc tslint prevent code about console 😢 
If you think it's enough to notice through the braking change, I'll remove it!

Comment (abhiomkar) [reply packages/mdc-linear-progress/constants.ts]: We don't have to throw warning. But we'll have to update existing codebase to use these new constants. :)

Comment (TroyTae) [reply packages/mdc-linear-progress/constants.ts]: Yes it looks to do that.
I'll refactor in the `feat/update_constants` branch separately from this PR :)

**Pull request closed without merging**